jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -3,7 +3,44 @@ local ksm = (import 'kube-state-metrics/kube-state-metrics.libsonnet') +
               name:: 'kube-state-metrics',
               namespace:: 'monitoring',
               version:: '1.9.4',  //$._config.versions.kubeStateMetrics,
-              image:: 'quay.io/coreos/kube-state-metrics',  //$._config.imageRepos.kubeStateMetrics + ':v' + $._config.versions.kubeStateMetrics,
+              image:: 'quay.io/coreos/kube-state-metrics:v' + self.version,  //$._config.imageRepos.kubeStateMetrics + ':v' + $._config.versions.kubeStateMetrics,
+              serviceMonitor:: {
+                apiVersion: 'monitoring.coreos.com/v1',
+                kind: 'ServiceMonitor',
+                metadata: {
+                  name: 'kube-state-metrics',
+                  namespace: ksm.namespace,
+                  labels: {
+                    'app.kubernetes.io/name': 'kube-state-metrics',
+                  },
+                },
+                spec: {
+                  jobLabel: 'k8s-app',
+                  selector: {
+                    matchLabels: {
+                      'app.kubernetes.io/name': 'kube-state-metrics',
+                    },
+                  },
+                  endpoints: [
+                    {
+                      port: 'http-metrics',
+                      interval: '30s',
+                      scrapeTimeout: '30s',
+                      honorLabels: true,
+                      relabelings: [
+                        {
+                          regex: '(pod|service|endpoint|namespace)',
+                          action: 'labeldrop',
+                        },
+                      ],
+                    },
+                    {
+                      port: 'telemetry',
+                      interval: '30s',
+                    },
+                  ],
+                },
+              },
             };
```

Comment:
We can just not make this a local variable and just export this object directly.

Reply:
fixed.